tools/azsdk-cli/auto-documentation.yml

```diff
@@ -0,0 +1,40 @@
+jobs:
+  - job: UpdateMCPDocumentation
+    displayName: 'Update MCP Tools Documentation'
+    pool:
+      name: $(LINUXPOOL)
+      image: $(LINUXVMIMAGE)
+      os: linux
+    
+    steps:
+    - checkout: self
+      displayName: 'Checkout current branch'
+    
+    - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+      parameters:
+        Repositories:
+          - Name: Azure/azure-sdk-tools
+            Commitish: main
+            WorkingDirectory: $(System.DefaultWorkingDirectory)/azure-sdk-tools-main
+        Paths:
+          - 'tools/azsdk-cli/**'
+    
+    - template: /eng/pipelines/templates/steps/install-dotnet.yml
+    
+    - task: DotNetCoreCLI@2
+      displayName: 'Build Azure SDK CLI (Release)'
+      inputs:
+        command: 'build'
+        projects: '$(System.DefaultWorkingDirectory)/tools/azsdk-cli/Azure.Sdk.Tools.Cli/Azure.Sdk.Tools.Cli.csproj'
+        arguments: '--configuration Release'
+    
+    - task: PowerShell@2
+      displayName: 'Generate MCP tools documentation'
```

Comment:
We can avoid building the exe using existing script that will download the exe.
1. Run ` pwsh ./eng/common/mcp/azure-sdk-mcp.ps1 -InstallDirectory  $(System.DefaultWorkingDirectory)`
2. Pass  $(System.DefaultWorkingDirectory)/azsdk as the executable path